pkg/lib/cockpit.js

```diff
@@ -1750,7 +1750,7 @@
         };
 
         function unclaim() {
-            if (source && source.close)
+            if (source?.close)
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18448-20230312-091152-46c0aa82-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)